### ISSUE TITLE
Incorrect version selected by github-release ice of jq on Linux

### ISSUE THREAD

**Issue by kirintwn** [zdharma/zinit#403]
**Describe the bug**

I'm using the config to install `jq` via zinit:
```sh
zinit ice from"gh-r" as"program" mv"jq* -> jq"
zinit light stedolan/jq
```
Which works fine and installed the latest version [`jq-1.6`](https://github.com/stedolan/jq/releases/tag/jq-1.6) on my Macbook (MacOS).

However, I have another machine which runs Ubuntu 20.04 with an AMD 3700x (amd64) CPU using the same zinit config.

The zinit installed [`jq-1.5rc2`](https://github.com/stedolan/jq/releases/tag/jq-1.5rc2) for this machine.

**To Reproduce**

Install `jq` with the following config on an amd64 based Linux machine:
```sh
zinit ice from"gh-r" as"program" mv"jq* -> jq"
zinit light stedolan/jq
```

**Expected behavior**

should installed [`jq-1.6`](https://github.com/stedolan/jq/releases/tag/jq-1.6) on an amd64 based Linux machine

**Screenshots**

![image](https://user-images.githubusercontent.com/11881669/91319300-cdd8ed80-e7ee-11ea-91b2-6f63bf4c12dd.png)

**Versions:**

 - $ZSH_VERSION == 5.8
 - Operating system name and version (uname -a): Linux main-desktop 5.4.0-42-generic #46-Ubuntu SMP Fri Jul 10 00:24:02 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

**Additional context**

I believe it's due to the naming of jq release changed from `jq-linux-x86_64` to `jq-linux64`, which cause the algorithm failed to choose the latest version. 

PR https://github.com/zdharma/zinit/pull/269 may be relevant.

**Comment by psprint** [comment]
Could you check again before I debug – the file might not have been released for Linux in that version at that time…

**Comment by kirintwn** [comment]
Hi, I'm sure that `jq 1.6` 's asset `jq-linux64' has been there since 2018, and I can reproduce the problem now by using linux amd64 machine with the zinit config above.

**Comment by xeho91** [comment]
I'm not sure if this should be related to this issue, but since this is about the `jq` tool and as well picking the wrong architecture.
It picks **x32** instead of **x64** from: https://github.com/stedolan/jq/releases/tag/jq-1.6

I have a problem to install the correct version of `jq` on my Raspberry Pi 4 (using 64bit OS).
![image](https://user-images.githubusercontent.com/18627568/97853996-c81cee00-1cf0-11eb-89d2-7b75f8171729.png)

**Comment by docwhat** [comment]
@xeho91 -- There isn't an `arm` version of `jq` available; they only have intel: [jq 1.6 release assets](https://github.com/stedolan/jq/releases/tag/jq-1.6).

Sorry, it's unrelated to this issue and should be brought up with the `jq` folks.

**Comment by docwhat** [comment]
FYI: PR #442 will fix this issue...
